examples/algorithms/plot_greedy_coloring.py

```diff
@@ -0,0 +1,40 @@
+"""
+=====================
+Greedy Coloring Algorithm
+=====================
+We attempt to color a graph using as few colors as possible, where no neighbours of a node can have same color as the node itself.We use the function nx.greedy_color that implements the greedy coloring algorithm for a given graph.The function returns a dictionary that maps each node to its assigned color.
```

Comment:
Just a couple nits - lines should generally be ~88 characters long and the heading bars should match the length of the title: take a look at some of the other examples for reference!

Reply:
I will keep this in mind from now on. I will refer to other examples too to understand the format better. Thanks!